### DISCUSSION TITLE
CS Tools Install checking for uninstalled python version

### DISCUSSION THREAD

**Discussion by IanMiner** [thoughtspot/cs_tools#182] [General] [answered]
I was running into the msgspec error that the fix was to uninstall python 3.13 and install 3.12 because the msgspec whl was compatible with that version of python but now my CS Tools install keeps checking for python 3.13 and failing even though I uninstalled it and removed it from my PATH. Any ideas on how to fix this?
<img width="862" alt="cstootserror" src="https://github.com/user-attachments/assets/2a7ef0b5-4276-4dbb-81df-2738bc335f7b">

**Comment by boonhapus** [comment, marked as answer]
@IanMiner can you try the `--reinstall` option ? It should nuke the virtual environment and reset the path variables in the process.

**Reply by IanMiner** [reply]
Like this?
powershell -c "IRM https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py | python - --reinstall"

**Reply by IanMiner** [reply]
ok that worked thank you very much
